### DISCUSSION TITLE
Kimai Calendar Inquiries

### DISCUSSION THREAD

**Discussion by seleondevel** [kimai/kimai#4971] [Plugins]
Hello.

If you look at the CalendarSourceType or CalendarSourceEvent code for the Kimai calendar event, you can only see individual users' schedules, and the Kimai calendar template also has a structure in which only your own users can be selected and viewed.

However, I would like to display not only the timesheet I have written on the Kimai calendar, but also the schedule of other users included in the attendance field that I have arbitrarily added.

@kevinpapst  Would it be impossible to implement these features in a plug-in that I created separately?

**Comment by kevinpapst** [comment]
Sure. Fetch the event and add your own calendar source.
Then the route for this new source can return whatever data you want.
